### PR TITLE
add AUS datacards

### PR DESCRIPTION
**Pull request by JRichetti**
Add shapefiles and data cards for Australia

### CODE REVIEW

File: data_preparation/crop_statistics_AUS/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Dataset owner
+Commonweath of Australia
+
+## Data card author
+Richetti, Jonathan
+
+## Dataset overview
+**Crops**: Wheat
+
+**Variables** [unit]: t / ha
```
Comment (ellaampy) [inline data_preparation/crop_statistics_AUS/README.md:26]: please here the variables available in the database and their unit e.g. yield [t/ha], production [t], area sown [ha]. The R script shows the database also contains production and area sown.

File: data_preparation/crop_statistics_AUS/From abares to wheat df for AgML.R [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+wheat_df = drop_na(wheat_df) %>% select(Year,ABARES.region,Yield)
+
+head(wheat_df)
+
+write.csv(wheat_df,'wheat_Australia.csv')
+
+ggplot(wheat_df)+
+  geom_line(aes(x=Year,y=Avg_Yield_tperha))+
+  facet_wrap(~ABARESregion)+
+  theme_bw()
```
Comment (ellaampy) [inline data_preparation/crop_statistics_AUS/From abares to wheat df for AgML.R:21]: @JRichetti could you please produce the final table according to the prescribed data format. kindly check an example table for India [here](https://github.com/BigDataWUR/AgML-crop-yield-forecasting/blob/main/data_preparation/crop_statistics_IN/data_preparation.ipynb) (at the end of the script)

**Review by ellaampy — CHANGES_REQUESTED** [review]
@JRichetti please address the comments on crop statistics readme and data preparation script

**Review by krsnapaudel — CHANGES_REQUESTED** [review]
There are some lines copied from the data card template and not filled in correctly.
Examples: Provenance and References.

Similarly, variables are not included.

For shapefiles data card. Please check a similar example from EU or US to include expected overview.

**Comment by JRichetti** [comment]
I'm not super savy with git, can you check if what I pushed is as expected now?

**Comment by krsnapaudel** [comment]
@JRichetti : ideally we would want the official shapefiles that include all admin boundaries up to the units with statistics.

File: data_preparation/shapefile_AU/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+# Australian Regions (ABARES)
```
Comment (krsnapaudel) [inline data_preparation/shapefile_AU/README.md:1]: The directory name should be `shapefiles_AU`.

File: data_preparation/shapefile_AU/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+## Data access API
+Not available.
+
+## Provenance 
+Farm survey definitions and methods used to produce these statistics are available on the ABARES website. For further analysis and information about the design of these statistics see Boult and Jackson (2019).
+
+Boult, C & Jackson, T 2019, ‘Disaggregating farm performance statistics by size, 2017–18’, in Agricultural Commodities: March quarter 2019, Australian Bureau of Agricultural and Resource Economics and Sciences, Canberra.
```
Comment (krsnapaudel) [inline data_preparation/shapefile_AU/README.md:45]: This seems copied from previous README. Please update.

**Comment by JRichetti** [comment]
I've updated the script and the readme files. I also uploaded the updated data and shapes with matching adm_id to the Google drive. Hope it is ok now.

**Review by krsnapaudel — APPROVED** [review]

**Comment by krsnapaudel** [comment]
Thanks @JRichetti. I will rename your script. Otherwise it's good.

**Comment by krsnapaudel** [comment]
@ellaampy : merging is blocked because of your review. Please see updates. Thanks.

**Review by ellaampy — APPROVED** [review]

**Pull request merged**